### ISSUE TITLE
Wrong discount calculation on bulk pricing

### ISSUE THREAD

**Issue by intoxstudio** [Freemius/wordpress-sdk#79]
<img width="586" alt="udklip" src="https://cloud.githubusercontent.com/assets/1677777/17568754/5835fec8-5f45-11e6-9639-9560b4644393.PNG">

Annual price per month for 1 license is $3.99 (20% discount). For 5 it's $16,95/m.

So firstly, it looks like the calculated multi-site license discount is an absolute value ($36). Otherwise it is correct.
I can't wrap my head around how the 160% is calculated.
Calculation should probably be Annual - (3,99 \* 5\*12) = $60
To get the percentage: $60 / Annual * 100 ~= 20%

IMO it would also make more sense for the user if those lines are reversed, ie. first the annual license discount, then the multi-site license discount.

Just to be clear, this issue does not change the subtotal, which is still correct (as entered in the dashboard).

SDK version 1.1.9.

**Comment by vovafeldman** [comment]
@intoxstudio do you also see the same discount miscalculation when testing **Freemius Checkout**?

**Comment by intoxstudio** [comment]
I don't see an option to select bulk pricing/tiers in the Checkout modal.

<img width="458" alt="udklip" src="https://cloud.githubusercontent.com/assets/1677777/17778195/800b1f5e-6563-11e6-8aa9-2e42c6c28e43.PNG">

Though, it looks like is has the same problem with absolute value in the discount.

**Comment by vovafeldman** [comment]
To test a multi-site license with the checkout add a ‘licenses‘ parameter to the JavaScript init with the relevant number. E.g. for 5-site license add:
‘licenses: 5,‘

**Comment by intoxstudio** [comment]
That is pretty neat! I tested it with this result:

<img width="464" alt="udklip" src="https://cloud.githubusercontent.com/assets/1677777/17859988/00775ef2-688c-11e6-85aa-9d988142adab.PNG">

So looks like even the first line is off here, not being multiplied by 5.

By the way, I tested it in Firefox on localhost, and I think the problem with the logo not being shown has to do with it being non-SSL. On https://dev.institute/wordpress/sidebars-pro the logo is displayed fine in all browsers (even IE). Not a big issue, so thought I'd just write it here.
